src/Avalonia.Controls/ItemsControl.cs

```diff
@@ -53,6 +53,12 @@ public class ItemsControl : TemplatedControl, IItemsPresenterHost, ICollectionCh
         public static readonly StyledProperty<IDataTemplate> ItemTemplateProperty =
             AvaloniaProperty.Register<ItemsControl, IDataTemplate>(nameof(ItemTemplate));
 
+        /// <summary>
+        /// Defines the <see cref="WrapSelection"/> property.
+        /// </summary>
+        public static readonly StyledProperty<bool> WrapSelectionProperty =
+            AvaloniaProperty.Register<ItemsControl, bool>(nameof(WrapSelection), defaultValue: false);
```

Comment:
should we set the default value to true?

Reply:
Probably not, I don't think most lists are wrapping by default usually?